Library/Formula/abcde.rb

```diff
@@ -12,17 +11,17 @@ class Abcde < Formula
     sha1 "b98cc145bbbae45421b36721f39f0a5a7a8ee3f7" => :mountain_lion
   end
 
-  depends_on 'cd-discid'
-  depends_on 'cdrtools'
-  depends_on 'id3v2'
-  depends_on 'mkcue'
-  depends_on 'flac' => :optional
-  depends_on 'lame' => :optional
-  depends_on 'vorbis-tools' => :optional
+  depends_on "cd-discid"
+  depends_on "cdrtools"
+  depends_on "id3v2"
+  depends_on "mkcue"
+  depends_on "flac" => :optional
+  depends_on "lame" => :optional
+  depends_on "vorbis-tools" => :optional
 
   def install
-    bin.install 'abcde', 'abcde-musicbrainz-tool', 'cddb-tool'
-    etc.install 'abcde.conf'
-    man1.install 'abcde.1', 'cddb-tool.1'
+    bin.install "abcde", "abcde-musicbrainz-tool", "cddb-tool"
+    etc.install "abcde.conf"
+    man1.install "abcde.1", "cddb-tool.1"
   end
 end
```

Comment:
@mikemcquaid I’d like to, but I really don’t know how to do that. I can’t even install `abcde` because it depends on `cd-discid`, which itself uses a patch from `trac.macports.org`, which is down at the moment ([they’re aware of that](https://twitter.com/macports/status/546710316315590656)). I just found it supports a [`-v` option](http://git.einval.com/cgi-bin/gitweb.cgi?p=abcde.git;a=blob;f=abcde.1;h=90929429025660d0b6ea1713c7cd4ca31436239d;hb=HEAD#l173) for the version, and added a test for that.